aten/src/ATen/Parallel.h

```diff
@@ -122,43 +72,17 @@ two partial results and an identity.
 [1] https://software.intel.com/en-us/node/506154
 */
 template <class scalar_t, class F, class SF>
-inline scalar_t parallel_reduce(
+scalar_t parallel_reduce(
```

Comment:
Are there any implications to not using inline?

Reply:
I need to double check, possibly I can put it back, will check